components/security/secure_tools.rst

```diff
@@ -1,47 +1,39 @@
-Securely Generating Random Numbers
+Securely Generating Random Strings 
 ==================================
 
 The Symfony Security component comes with a collection of nice utilities
 related to security. These utilities are used by Symfony, but you should
 also use them if you want to solve the problem they address.
 
-Generating a Secure random Number
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Generating a Secure random
```

Comment:
People might search for "random number symfony" on search engines though. So it does make sense to me to keep "number" in there. Or else "a secure random string or number".